submissions/drewzag/memory-pair-game/app.js

```diff
@@ -0,0 +1,87 @@
+const section = document.querySelector('section')
+
+const getData = () => [
+  { imgSrc: './images/ari.webp', id: 'arisona' },
+  { imgSrc: './images/cle.webp', id: 'clevlend' },
+  { imgSrc: './images/lv.webp', id: 'las-vegas' },
+  { imgSrc: './images/mia.webp', id: 'miami' },
+  { imgSrc: './images/sea.webp', id: 'seattle' },
+  { imgSrc: './images/tb.webp', id: 'tampa-bay' },
+]
+
+const randomize = () => {
+  const cardData = getData().concat(getData())
+  cardData.sort(() => Math.random() - 0.5)
+  return cardData
+}
+
+const cardGenerator = () => {
+  const cardData = randomize()
+
+  const cardsContainer = document.createDocumentFragment()
+  cardData.forEach((item) => {
+    const card = document.createElement('div')
+    const face = document.createElement('img')
+    const back = document.createElement('div')
+    card.classList = 'card'
+    card.setAttribute('name', item.id)
+    face.classList = 'face'
+    back.classList = 'back'
+
+    face.src = item.imgSrc
+
+    card.append(face, back)
+    cardsContainer.appendChild(card)
+
+    card.addEventListener('click', ({ target }) => {
+      card.classList.toggle('toggle')
+      checkCards(target)
+    })
+  })
+  section.appendChild(cardsContainer)
+}
+
+const checkCards = (target) => {
+  const clickedCards = target
+  clickedCards.classList.toggle('flipped')
+  const flippedCards = document.querySelectorAll('.flipped')
+  const cards = document.querySelectorAll('.card')
+  const toggeledCards = document.querySelectorAll('.toggle')
+
+  if (flippedCards.length === 2) {
+    if (flippedCards[0].getAttribute('name') === flippedCards[1].getAttribute('name'))
+      flippedCards.forEach((card) => {
+        card.classList.remove('flipped')
+      })
+    else {
+      flippedCards.forEach((card) => {
+        card.classList.remove('flipped')
+        setTimeout(() => card.classList.remove('toggle'), 1000)
+      })
+    }
+  }
+
+  if (cards.length === toggeledCards.length) {
+    setTimeout(() => restart(), 1000)
```

Comment:
Please check what arguments need to pass to setTimeout and make a fix here
https://www.w3schools.com/jsref/met_win_settimeout.asp